### ISSUE TITLE
An error in DGLGraph().from_networkx() with edge features

### ISSUE THREAD

**Issue by qiaochen** [dmlc/dgl#283]
## To Reproduce

```
import networkx as nx

nxg = nx.DiGraph()
nxg.add_nodes_from([(1,2,3)],features=[[2,3],[1,4],[5,9]])
nxg.add_edge(1, 2, features=[3,4])
dglg = DGLGraph()
dglg.from_networkx(nxg, edge_attrs=['features'])
```

```
  File "/home/chen/anaconda3/lib/python3.6/site-packages/dgl/graph.py", line 1187, in from_networkx
    attr_dict[key][eid] = attrs[key]

UnboundLocalError: local variable 'attrs' referenced before assignment
```

## Expected behavior

Graph conversion  successfully

## Environment

 - DGL Version (0.1.2):
 - Backend Library & Version (ePyTorch 0.4.1):
 - OS (Ubuntu 18.04):
 - How you installed DGL (conda):
 
## Additional context
This error should be due to the variable `attr`  that is not aligned with the `attrs` variable at the bottom line
```
for eid, (_, _, attr) in enumerate(nx_graph.edges(data=True)):
                    for key in edge_attrs:
                        attr_dict[key][eid] = attrs[key]
```
https://github.com/dmlc/dgl/blob/322bd713e4b0ce90fae12292c90d45710af78515/python/dgl/graph.py#L1185
```
```

**Comment by ylfdq1118** [comment]
Thanks for reporting the bug. We will fix it soon.

**Comment by ylfdq1118** [comment]
This bug should have been fixed in release 0.1.3.
